jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -292,17 +349,38 @@ private void testGetOfColInfo(
     }
   }
 
+  /**
+   * Validate resultSet MetaData returned from getSchemas.
+   *
+   * @param resultSet ResultSet need to be validated.
+   * @throws SQLException If an error occurs while retrieving the value.
+   */
+  private void testGetSchemasResult(ResultSet resultSet) throws SQLException {
+    final String[] string1 = {"", "testDB", null};
+    final List<String[]> strings = new ArrayList<>();
+    strings.add(string1);
+
+    int numRows = 0;
+    while (resultSet.next()) {
+      for (int i = 1; i <= resultSet.getMetaData().getColumnCount(); ++i) {
+        Assertions.assertEquals(strings.get(numRows)[i], resultSet.getString(i));
+      }
+      numRows++;
+    }
+    Assertions.assertEquals(1, numRows);
+  }
+
   /**
    * Validate resultSet MetaData returned from getColumns.
    *
    * @param resultSet ResultSet need to be validated.
    * @throws SQLException If an error occurs while retrieving the value.
    */
   private void testGetColumnsResult(ResultSet resultSet) throws SQLException {
-    final String[] string1 = {"", "testDB", null, "testTable", "ColName", "12", "UNKNOWN",
+    final String[] string1 = {"", null, "testDB", "testTable", "ColName", "12", "UNKNOWN",
```

Comment:
Add test cases for 
1/ Empty ResultSet
2/ expecting only one row in resultset
3/ multiple rows to simulate paginated results when metadata query is run against Timestream